test/op_shape_test.cpp

```diff
@@ -966,6 +966,30 @@ TEST_CASE(broadcast_for_dot_dyn2)
         s0);
 }
 
+TEST_CASE(broadcast_with_dims0)
```

Comment:
@CharlieL7 I think these two tests were from you, but Github is showing them as new to this PR.  Did you mean to keep them, or not?

Reply:
The tests look the same to me, just moved around.